### ISSUE TITLE
TSC Infrastructure Meeting Minutes 27 April 2021

### ISSUE THREAD

**Issue by bobmcc9** [genevaers/community#127]
**SAFR Production Binary Deliverable to GIT**

- license.md file has been created for this particular deliverable
- readme.md has been updated to hold the deployment instructions for this Repo
- currently on Hold waiting for an examination of candidate PDS's and contents of those PDS's
- Kip asked about RM issues in this license.md  Bobmcc to chase up

**MR91 Binary Deliverable**

- approval received to include SAX XML Parser modules from XML Toolkit team in China
- deployment into z/OS from Git are understood
- as source for this deliverable will not reside in GIT only a binary file, we need to perform a special build on KANPLEX to ensure correct version release number
- bobmcc advised a bespoke build for this with a specific VRM needed

**Performance Engine sans CPP code Build**

- we have a deployment plan for this Repo
- after GIT cloning to USS, we run a shell script to create and populate the source PDS's on z/OS
- then a supplied batch job (in SAMP) uses a supplied REXX pgm (in SAMP) to create 3 build jobs
- BUILD1 to assemble all parts listed in INVM 
- BUILD2 to preprocess and assemble all DB2 assemble files in INVM
- BUILD3 to bind all modules as listed in INVM
- VICOM have the latest Toolkit structured macros
- Kip asked that POLYCEPHALY be looked as well for builds

**Move Performance Engine to IBM Internal GITHUB**

- currently waiting on a INVM examination for open source shipment

**Installation Verification**

- Plan here is to deploy the testsuite
- on USS on Kanplex we performed a GIT clone of the Test Framework repo
- we deployed GRADLE software onto USS
- Using GRADLE we ran the Test Framework as a pure java application to perform all the tests
- Took under 10 minutes
- Issue does exist where the ouput HTML is sent/stored

**Java Source Scans**

- some training exist to show SAFR staff to use SONAR QUbe
-  We use the community edition of SONAR QUBE 

**Move WorkBench to external Git**

- waiting on IBM development staff for final tweaks

**Move Test FrameWork to external Git**

- waiting on development tweaks for this repo

**Kanban**

- update GD 24 with results of testing Test Framework from USS
- update GD 18 with data about deploying PE

**Comment by KipTwitchell** [comment]
Meeting complete
